qwc-front.php

```diff
@@ -60,6 +60,27 @@ function qwc_add_filters_front() {
 	//foreach ( $url_filters as $name => $priority ) {
 	//	add_filter( $name, 'qtranxf_convertURL', $priority );
 	//}
+
+	/* Fix the product categories and tags (displayed above the "additional informations" tab) */
+	add_filter( 'wp_get_object_terms', function ( $terms ) {
+		foreach ( $terms as $term ) {
+			if ( $term->taxonomy == 'product_cat' || $term->taxonomy == 'product_tag' ) {
```

Comment:
This is what I see when I disable that part of the patch:
![image](https://cloud.githubusercontent.com/assets/362092/6422503/b7402e4c-bed9-11e4-8082-87e70604ccff.png)

The category and tags lists below the attributes (single-product view) are not translated.